src/action-ros-ci.ts

```diff
@@ -214,7 +215,11 @@ async function run() {
 		// rosdep does not reliably work on Windows, see
 		// ros-infrastructure/rosdep#610 for instance. So, we do not run it.
 		if (!isWindows) {
-			await execBashCommand("rosdep update --include-eol-distros");
+			retry(async () => {
```

Comment:
I think this needs to use `await` so that we actually wait for the installation to finish. As you can see in some of the current tests, the rosdeps were not installed before the `colcon build` is attempted

```suggestion
			await retry(async () => {
```

Reply:
added await before retry